pkg/errmetrics/metrics.go

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package errmetrics
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/cilium/tetragon/pkg/bpf"
+	"github.com/cilium/tetragon/pkg/logger"
+	"github.com/cilium/tetragon/pkg/logger/logfields"
+	"github.com/cilium/tetragon/pkg/metrics"
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+)
+
+var errMetricsMap Map
+
+var errorMetricsMetric = metrics.MustNewCustomCounter(
+	metrics.NewOpts(
+		consts.MetricsNamespace, "error_metrics", "error_total",
```

Comment:
Q: do we want to somehow highlights these metrics come from `bpf`?
Like setting `bpf` as subsystem:
```suggestion
		consts.MetricsNamespace, "bpf", "error_metrics_total",
```

Reply:
yeah good idea, error_metrics stutters and is not great, let's use bpf!